client/src/tests/MovieContainer.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { mount } from 'enzyme';
+import MovieContainer from '../components/MoviesContainer';
+import movies from '../components/movies';
+
+it ('renders correctly', ()=>{
+    const component = renderer.create(<MovieContainer />);
```

Comment:
_trivial_

Since you've already been using `enzyme` you can take advantage of doing snapshot testing by means of `enzyme` entirely(without using `react-test-renderer`) e.g.

```javascript
const wrapper = mount(<MovieContainer />);
expect(wrapper).toMatchSnapshot();
```